onnx/defs/tensor/defs.cc

```diff
@@ -158,3 +158,40 @@ Takes a  parameter `axes` with a list of axes to squeeze.
 )DOC")
     .Input(0, "data", "Tensors with at least max(dims) dimensions.")
     .Output(0, "squeezed", "Reshaped tensor with same data as input.");
+
+OPERATOR_SCHEMA(Pad)
+    .NumInputs(1)
+    .NumOutputs(1)
+    .Attr("paddings",
+          "List of integers indicate the padding sizes, paddings's length should be the double of input's dimension.",
```

Comment:
you need to describe in which order the pads are listed. Is it axis_0_begin, axis_0_end, axis_1_begin, ... ?

Reply:
Yeah, I will add it to the doc.